x-pack/plugins/apm/public/components/shared/search_bar.tsx

```diff
@@ -28,6 +31,42 @@ interface Props {
   kueryBarBoolFilter?: QueryDslQueryContainer[];
 }
 
+function ApmDatePicker() {
+  const { query } = useApmParams('/*');
+
+  if (!('rangeFrom' in query)) {
+    throw new Error('range not available in route parameters');
+  }
+
+  const {
+    rangeFrom,
+    rangeTo,
+    refreshPaused: refreshPausedFromUrl = 'true',
+    refreshInterval: refreshIntervalFromUrl = '0',
+  } = query;
+
+  const refreshPaused = refreshPausedFromUrl
+    ? toBoolean(refreshPausedFromUrl)
+    : undefined;
```

Comment:
do you really need this check? you're adding a default value to `refreshPausedFromUrl` o line 44, so it'll never be undefined, right?

Reply:
fixed, thanks!